clients/src/main/resources/common/message/TxnOffsetCommitRequest.json

```diff
@@ -20,7 +20,9 @@
   // Version 1 is the same as version 0.
   //
   // Version 2 adds the committed leader epoch.
-  "validVersions": "0-2",
+  //
+  // Version 3 adds the member.id, group.instance.id and generation.id.
+  "validVersions": "0-3",
   "flexibleVersions": "none",
```

Comment:
Since TxnOffsetCommit has been converted to use the generated protocol and we are bumping it here, let's add flexible version support. I was going to do this in #7931, but we may as well just do one version bump instead of two.